src/app/components/planner-list/planner-list.component.html

```diff
@@ -199,45 +202,84 @@ <h4>Please wait, we are loading your data.</h4>
                 </a>
               </ng-template>
           </ngx-datatable-column>
-          <!-- columns id-assignees -->
-          <ngx-datatable-column
-            *ngFor="let c of (columns | filterColumn:'display')"
-            [isTreeColumn]="c.name==='Title'"
-            [name]="c.name"
-            [resizeable]="false"
-            [width]="c.width"
-            [sortable]="c.name === 'Title'"
-            [cellClass]="'cell-class'"
-            [headerClass]="'header-cell'">
-              <ng-template ngx-datatable-cell-tree-expander *ngIf="c.name==='Title'">
-                <i class="fa fa-angle-right tree-icon"></i>
-              </ng-template>
+          <!-- columns id-assignees-->
+            <ngx-datatable-column
+              *ngFor="let c of (columns | filterColumn:'display')"
+              [isTreeColumn]="c.name === 'Title'"
+              [name]="c.name"
+              [resizeable]="false"
+              [width]="c.width"
+              [sortable]="c.name === 'Title'"
+              [cellClass]="'cell-class'"
+              [headerClass]="'header-cell'">
+                <ng-template ngx-datatable-cell-tree-expander *ngIf="c.name==='Title'">
```

Comment:
Give space before and after ``` === ```. As you did it in line number 208. Do the same to maintain the consistency throughout the code.